webrtc.html

```diff
@@ -2251,6 +2251,10 @@
             state. Note the local or remote SDP descriptions in the
 	    previous stable state could be null if there has
             not yet been a successful offer-answer negotiation.</p>
+
+            <p>If the value of a <code>RTCRtpTransceiver.mid</code> was set to a non-null value by
```

Comment:
We have made a few runs to try to get rid of the "static like" syntax when referring to properties. Please don't add new ones.